frontend/lib/models/test_execution.dart

```diff
@@ -57,7 +57,7 @@ enum TestExecutionStatus {
       case inProgress:
         return const Icon(YaruIcons.refresh, size: size);
       case passed:
-        return const Icon(YaruIcons.ok, color: YaruColors.success, size: size);
+        return Icon(YaruIcons.ok, color: YaruColors.light.success, size: size);
```

Comment:
dumb Q: are we Ok with not checking the theme here?

Reply:
Both dark and light themes have the same color for success. But mainly, our dashboard doesn't support different themes. i.e. you'll see the same colors irrelevant of what you pick